include/dd/Package.hpp

```diff
@@ -542,12 +554,14 @@ namespace dd {
 
         UniqueTable<vNode, UT_VEC_NBUCKET, UT_VEC_INITIAL_ALLOCATION_SIZE> vUniqueTable{nqubits};
         UniqueTable<mNode, UT_MAT_NBUCKET, UT_MAT_INITIAL_ALLOCATION_SIZE> mUniqueTable{nqubits};
+        UniqueTable<dNode, UT_MAT_NBUCKET, UT_MAT_INITIAL_ALLOCATION_SIZE> dUniqueTable{nqubits};
```

Comment:
why aren't there separate template parameters for the density matrix unique table, while there are some for the compute table?

Reply:
Done. I added separate template parameters for density matrix unique tables